swagger2markup-swagger-v3/src/main/java/io/github/swagger2markup/OpenApiPathsSection.java

```diff
@@ -0,0 +1,37 @@
+package io.github.swagger2markup;
+
+import io.github.swagger2markup.adoc.ast.impl.SectionImpl;
+import io.swagger.v3.oas.models.Paths;
+import org.asciidoctor.ast.Document;
+
+import java.util.Optional;
+
+import static io.github.swagger2markup.OpenApiHelpers.*;
+import static io.github.swagger2markup.OpenApiServerSection.appendServersSection;
+
+public class OpenApiPathsSection {
+
+    public static void appendPathSection(Document document, Paths apiPaths) {
+        if(null == apiPaths || apiPaths.isEmpty()) return;
+
+        SectionImpl allPathsSection = new SectionImpl(document);
+        allPathsSection.setTitle(SECTION_TITLE_PATHS);
+
+        apiPaths.forEach((name, pathItem) -> {
+            pathItem.readOperationsMap().forEach(((httpMethod, operation) -> {
+                SectionImpl operationSection = new SectionImpl(allPathsSection);
+                String summary = Optional.ofNullable(operation.getSummary()).orElse("");
+                operationSection.setTitle((italicUnconstrained(httpMethod.name().toUpperCase()) + " " + monospaced(name) + " " + summary).trim());
```

Comment:
Can we change this to how it looks today?
![image](https://user-images.githubusercontent.com/5815046/72421087-27b90580-3780-11ea-8917-c494586a76d6.png)

Reply:
Certainly can be done, I opted for the current implementation as summary is an optional value can be missing. what could be used for such scenarios?